### DISCUSSION TITLE
can i ask copilot to answer through program like API call?

### DISCUSSION THREAD

**Discussion by tech2hard** [community/community#61976] [Copilot Conversations]
### Select Topic Area

General

### Body

How do i use copilot in C# code execution, like API call to open AI?

Has anyone went thru scenario on making AI call from executable and not at the programming stage?

**Comment by samus-aran** [comment]
Hi @tech2hard 

There isn't an Copilot API currently available. [This discussion ](https://github.com/orgs/community/discussions/61742)has a similar request. 

> Thank you sharing your feedback. I have asked my team internally to gather more details about any plans to provide open source API for Copilot.
